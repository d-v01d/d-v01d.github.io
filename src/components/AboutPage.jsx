import React, { Component } from 'react'
import Social from './Social'

export default class About extends Component {
  render() {
    return (
      <div id="home">
        <h2>Hi I'm Jason</h2>
        <p>Place Holder Text</p>
        <Social />
      </div>
    )
  }
}
