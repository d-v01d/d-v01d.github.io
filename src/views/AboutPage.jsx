import React from 'react'
import Social from '../components/Social'
import AboutJson from '../components/AboutJson'

export default () => (
  <div id="about">
    <AboutJson />
    {/* <Social /> */}
  </div>
)
