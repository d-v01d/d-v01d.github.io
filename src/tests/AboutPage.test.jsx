import React from 'react'
import { mount } from 'enzyme'
import AboutPage from '../views/AboutPage'

test('should render AboutPage correctly', () => {
  const wrapper = mount(<AboutPage />)
  expect(wrapper).toMatchSnapshot()
})
