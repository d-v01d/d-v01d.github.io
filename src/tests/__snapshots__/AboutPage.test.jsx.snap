// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AboutPage correctly 1`] = `
<div
  id="about"
>
  <_default />
  <Social />
</div>
`;
